module/files/helper.js

```diff
@@ -152,17 +155,18 @@ module.exports = class FilesHelper {
         const signedUrlsPromises = fileNames.map(async (fileName) => {
           let file =
             folderPath && folderPath !== "" ? folderPath + fileName : fileName;
-            let response = {
-              file: file,
-              payload: { sourcePath: file },
-              cloudStorage: cloudStorage.toUpperCase(),
-            };
-            
+          let response = {
+            file: file,
+            payload: { sourcePath: file },
+            cloudStorage: cloudStorage.toUpperCase(),
+          };
+          if (process.env.CLOUD_STORAGE_PROVIDER !== "gcloud") {
```

Comment:
why is this still needed ? @praveenKDass

Reply:
Yes its need ankit we can't call getDownloableUrl for GCP before uploading the file it it will throw error .once the object is present in bucket then only in GCP we can able to generate the url @ankit1shahu